build/publishNewVersion.mjs

```diff
@@ -74,7 +74,7 @@ const outputProcess = (process) => {
 
```

Comment:
I cannot comment on line 72, but it should be `const currentVersion = lastTag.replace(VERSION_PREFIX, '');`